package.json

```diff
@@ -89,10 +89,10 @@
     "@commitlint/config-conventional": "19.5.0",
     "@eslint/js": "~9.19.0",
     "@faker-js/faker": "9.4.0",
-    "@nestjs/cli": "11.0.2",
-    "@nestjs/schematics": "11.0.0",
-    "@nestjs/testing": "11.0.5",
-    "@nx/angular": "20.3.3",
+    "@nestjs/cli": "11.0.7",
+    "@nestjs/schematics": "11.0.5",
+    "@nestjs/testing": "11.1.3",
+    "@nx/angular": "21.2.0",
```

Comment:
Have you checked whether we need Nx migrations for this? Unfortunately since Nx keeps all its packages on the same versions, it's generally best to let Nx bump and migrate everything at once, though that can cause dependency issues with other packages. Going to try a migration now and see if anything breaks.